examples/api/lib/material/text_field/text_field.2.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for TextField
```

Comment:
If we have other examples for TextField, then we might want to call this out as a sample for Material Design 3 TextFields.

Reply:
Done